modules/ROOT/pages/about-administration-tasks.adoc

```diff
@@ -3,13 +3,13 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Anypoint Exchange provides those with the Exchange Administrator capabilities not shared with Exchange Contributors or Exchange Viewers.
+Anypoint Exchange provides those with the Exchange Administrator role capabilities not shared with Exchange Contributors or Exchange Viewers.
 
 You can perform the following tasks as an Exchange Administrator:
 
 * xref:to-set-permissions.adoc[Set Exchange role permissions] - Determine who in your organization should be an Exchange Administrator, Contributor, or Viewer. You can also provide role access for individual assets.
 * xref:to-save-searches.adoc[To Save Searches] - Save and maintain searches for assets within a business group.
-* xref:to-share-an-asset-with-a-user.adoc[To Share an Asset] - Enable another user in your organization to access this asset. You can also use Share to publish an API asset to a Public Portal.
+* xref:to-share-an-asset.adoc[To Share an Asset] - Enable users in your organization or external teams in another trusted organization to access this asset. You can also use Share to publish an API asset to a Public Portal.
```

Comment:
"Enable users in your organization or another external organization(with which the asset admin has shared the asset with) to access this asset..... "

We are not introducing the concept of teams in this release. We will have to change the docs again, when Trusted Org & internal teams are introduced later.

@mcampo .... FYI

Reply:
Got it. Fixed.